### PR TITLE
DRILL-7530: Fix class names in loggers

### PR DESCRIPTION
**Pull request by arina-ielchiieva**
[DRILL-7530](https://issues.apache.org/jira/browse/DRILL-7530): Fix class names in loggers

1. Fix incorrect class names for loggers.
2. Minor code cleanup.

## Testing
Run all tests.

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java [hunk 1]
```diff
@@ -603,7 +606,7 @@ public static boolean guessRows(RelNode rel) {
             || (tableMetadata = table.getGroupScan().getTableMetadata()) == null
             || !TableStatisticsKind.HAS_DESCRIPTIVE_STATISTICS.getValue(tableMetadata);
       } catch (IOException e) {
-        RelOptPlanner.LOGGER.debug("Unable to obtain table metadata due to exception:", e);
+        logger.debug("Unable to obtain table metadata due to exception: {}", e.getMessage(), e);
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java]: @vvysotskyi is this change ok? Or `RelOptPlanner` was used intensionally?

**Comment by cgivre** [comment]
LGTM +1.  @arina-ielchiieva Thanks for doing this.   Pretty surprising how many permutations of the logger there were.

File: exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java (hunk 1 above)
Comment (vvysotskyi) [reply exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java]: Yes, using a specific logger is more suitable. Thanks for fixing this!

**Comment by cgivre** [comment]
I could write a python script to do that.  
-- C

**Comment by vvysotskyi** [comment]
I believe there is some way to do it using IDE, so the order of the imports would be consistent.

**Comment by arina-ielchiieva** [comment]
@paul-rogers 

{quote}
Any plans to fix the remaining fully-qualified log class names? It gets tedious to do those by hand each time I touch a file.
{quote}
No, from my side. I also get tedious of the fixes but don't have time to fix all of them.

**Pull request closed without merging**